syntax = "proto3";
package auth;


message RegisterRequest {
    string login = 1;
    string password = 2;
}

message RegisterResponse {
    string token = 1;
}

message LogInRequest {
    string login = 1;
    string password = 2;
}

message LogInResponse {
    string token = 1;
}

service AuthServer {
    rpc Register(RegisterRequest) returns (RegisterResponse) {}
    rpc LogIn(LogInRequest) returns (LogInResponse) {}
}
